### PR TITLE
Replace snackbars

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/pages/jails/jail-list/jail-list.component.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ export class JailListComponent implements OnInit {
                 }
               }
               if (message === "") {
-                this.snackBar.open(T("Jails started."), 'close', { duration: 5000 });
+                this.dialogService.Info(T('Jails Started'), T("Jails started."));
```
Comment (erinix) [inline src/app/pages/jails/jail-list/jail-list.component.ts]: I would make the second "Jails started" just be an empty '', give it an 'info' icon and set html to true

File: src/app/helptext/sharing/iscsi/iscsi.ts [hunk 2]
```diff
@@ -156,8 +156,10 @@ export const helptext_sharing_iscsi = {
   globalconf_dialog_message: T("Enable this service?"),
   globalconf_dialog_button: T("Enable Service"),
 
-  globalconf_snackbar_message: T("Service started"),
-  globalconf_snackbar_close: T("close"),
+  globalconf_start_service_dialog: {
+    titlle: T('Service started'),
```
Comment (Mrt134) [inline src/app/helptext/sharing/iscsi/iscsi.ts]:
```suggestion
    title: T('Service started'),
```

File: src/app/pages/jails/jail-list/jail-list.component.ts [hunk 3]
```diff
@@ -236,7 +235,7 @@ export class JailListComponent implements OnInit {
         if (this.toActivatePool) {
           this.entityList.getData();
         }
-        this.entityList.snackBar.open("Successfully activate pool " + this.selectedPool , 'close', { duration: 5000 });
+        this.entityList.dialogService.Info(T('Jail Activated'), "Successfully activate pool <i>" + this.selectedPool + "</i>", '500px', 'info', true);
```
Comment (Mrt134) [inline src/app/pages/jails/jail-list/jail-list.component.ts]:
```suggestion
        this.entityList.dialogService.Info(T('Jail Activated'), "Pool <i>" + this.selectedPool + "</i> is active.", '500px', 'info', true);
```
grammo in original text, suggested small rework

File: src/app/pages/jails/storages/storage-list/storage-list.component.ts [hunk 4]
```diff
@@ -92,7 +91,7 @@ export class StorageListComponent {
       ]
     ]).subscribe((res) => {
       if (res[0] && res[0].state == 'up') {
-        this.snackBar.open(this.jailId + " should not be running when deleting a mountpoint.", 'close', { duration: 5000 });
+        this.dialog.Info(T('Delete Mountpoint'), "<i>" + this.jailId + T("</i> should not be running when deleting a mountpoint."), '500px', 'info', true);
```
Comment (Mrt134) [inline src/app/pages/jails/storages/storage-list/storage-list.component.ts]:
```suggestion
        this.dialog.Info(T('Delete Mountpoint'), "<i>" + this.jailId + T("</i> cannot be running when deleting a mountpoint."), '500px', 'info', true);
```
Is the dialog offering a suggestion or communicating an error?

File: src/app/pages/jails/storages/storage-list/storage-list.component.ts [hunk 5]
```diff
@@ -133,7 +132,7 @@ export class StorageListComponent {
       ]
     ]).subscribe((res) => {
       if (res[0] && res[0].state == 'up') {
-        this.snackBar.open(this.jailId + " should not be running when adding a mountpoint.", 'close', { duration: 5000 });
+        this.dialog.Info(T('Add Mountpoint'), "<i>" + this.jailId + "</i> should not be running when adding a mountpoint.", '500px', 'info', true);
```
Comment (Mrt134) [inline src/app/pages/jails/storages/storage-list/storage-list.component.ts]:
```suggestion
        this.dialog.Info(T('Add Mountpoint'), "<i>" + this.jailId + "</i> cannot be running when adding a mountpoint.", '500px', 'info', true);
```
As above

File: src/app/pages/plugins/plugins.component.ts [hunk 6]
```diff
@@ -253,8 +251,7 @@ export class PluginsComponent {
             entityDialog.dialogRef.close(true);
             self.entityList.loaderOpen = true;
             self.entityList.getData();
-
-            self.snackBar.open("Successfully activate pool " + value['selectedPool'] , 'close', { duration: 5000 });
+            self.dialogService.Info(T('Pool actived'), T("Successfully activate pool ") + value['selectedPool']);
```
Comment (Mrt134) [inline src/app/pages/plugins/plugins.component.ts]:
```suggestion
            self.dialogService.Info(T('Pool Activated'), T("Successfully activated pool ") + value['selectedPool']);
```

File: src/app/pages/sharing/iscsi/fibre-channel-ports/fibre-channel-port/fibre-channel-port.component.ts [hunk 7]
```diff
@@ -142,7 +143,7 @@ export class FibreChannelPortComponent implements OnInit {
         this.ws.call('fcport.update', [this.config.id, value]).subscribe(
             (res) => {
                 this.parent.loader.close();
-                this.parent.snackBar.open("Fiber Channel Port " + this.config.name + " update successful.", 'close', { duration: 5000 });
+                this.parent.dialogService.Info(T('Updated'), T('Fiber Channel Port ') + this.config.name + ' update successful.');
```
Comment (Mrt134) [inline src/app/pages/sharing/iscsi/fibre-channel-ports/fibre-channel-port/fibre-channel-port.component.ts]:
```suggestion
                this.parent.dialogService.Info(T('Updated'), T('Fibre Channel Port ') + this.config.name + ' update successful.');
```
I believe this is the industry standard spelling.

File: src/app/pages/sharing/iscsi/globalconfiguration/globalconfiguration.component.ts [hunk 8]
```diff
@@ -83,7 +82,7 @@ export class GlobalconfigurationComponent {
               this.ws.call('service.update', [service['id'], { enable: true }]).subscribe((updateRes) => {
                 this.ws.call('service.start', [service.service]).subscribe((startRes) => {
                   this.loader.close();
-                  this.snackBar.open(helptext_sharing_iscsi.globalconf_snackbar_message, helptext_sharing_iscsi.globalconf_snackbar_close);
+                  this.dialogService.Info(helptext_sharing_iscsi.globalconf_start_service_dialog.titlle, helptext_sharing_iscsi.globalconf_start_service_dialog.content);
```
Comment (Mrt134) [inline src/app/pages/sharing/iscsi/globalconfiguration/globalconfiguration.component.ts]:
```suggestion
                  this.dialogService.Info(helptext_sharing_iscsi.globalconf_start_service_dialog.title, helptext_sharing_iscsi.globalconf_start_service_dialog.content);
```

File: src/app/pages/task-calendar/cloudsync/cloudsync-list/cloudsync-list.component.ts [hunk 9]
```diff
@@ -130,9 +127,7 @@ export class CloudsyncListComponent implements InputTableConf {
           if (res) {
             this.ws.call('cloudsync.abort', [row.id]).subscribe(
               (wsRes) => {
-                this.translateService.get("close").subscribe((close) => {
-                  this.entityList.snackBar.open(T('Cloud sync stopped.'), close, { duration: 5000 });
-                });
+                  this.dialog.Info(T('Task Stoped'), T('Cloud sync <i>') + row.description + T('</i> stopped.'), '500px', 'info', true);
```
Comment (Mrt134) [inline src/app/pages/task-calendar/cloudsync/cloudsync-list/cloudsync-list.component.ts]:
```suggestion
                  this.dialog.Info(T('Task Stopped'), T('Cloud sync <i>') + row.description + T('</i> stopped.'), '500px', 'info', true);
```

File: src/app/pages/task-calendar/cloudsync/cloudsync-list/cloudsync-list.component.ts [hunk 10]
```diff
@@ -106,9 +105,7 @@ export class CloudsyncListComponent implements InputTableConf {
             row.state = 'RUNNING';
             this.ws.call('cloudsync.sync', [row.id]).subscribe(
               (res) => {
-                this.translateService.get("close").subscribe((close) => {
-                  this.entityList.snackBar.open(T('Cloud sync has started.'), close, { duration: 5000 });
-                });
+                this.dialog.Info(T('Task started'), T('Cloud sync <i>') + row.description + T('</i> has started.'), '500px', 'info', true);
```
Comment (Mrt134) [inline src/app/pages/task-calendar/cloudsync/cloudsync-list/cloudsync-list.component.ts]:
```suggestion
                this.dialog.Info(T('Task Started'), T('Cloud sync <i>') + row.description + T('</i> has started.'), '500px', 'info', true);
```

**Comment by lola27** [comment]
thanks @Mrt134

**Pull request merged**